lib/resolvers/event_project_mutations/removeProject.js

```diff
@@ -8,7 +8,9 @@ const removeEventProject = async (parent, args, context) => {
   const user = await User.findOne({ _id: context.userId });
   if (!user) {
     throw new NotFoundError(
-      requestContext.translate('user.notFound'),
+      process.env.NODE_ENV !== 'production'
```

Comment:
import the constant variables from the constants file and utilize them instead of the string here